### ISSUE TITLE
Error when running Publish Gulp task

### ISSUE THREAD

**Issue by tidusjar** [tidusjar/Ombi#1386]
Getting the following error when running publish in the .net core project

```
Process terminated with code 1.
events.js:160
      throw er; // Unhandled 'error' event
      ^
GulpUglifyError: unable to minify JavaScript
    at createError (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\gulp-uglify\lib\create-error.js:6:14)
    at wrapper (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\gulp-uglify\node_modules\lodash\_createHybrid.js:87:15)
    at trycatch (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\gulp-uglify\minifier.js:26:12)
    at DestroyableTransform.minify [as _transform] (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\gulp-uglify\minifier.js:79:19)
    at DestroyableTransform.Transform._read (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:147:83)
[01:39:38] Finished 'lib' after 21 s
[01:39:38] Finished 'npm' after 21 s
    at doWrite (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:345:64)
    at writeOrBuffer (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:334:5)
    at DestroyableTransform.Writable.write (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:271:11)
    at DestroyableTransform.ondata (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_readable.js:546:20)
    at emitOne (events.js:96:13)
    at DestroyableTransform.emit (events.js:188:7)
    at readableAddChunk (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_readable.js:217:18)
    at DestroyableTransform.Readable.push (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_readable.js:176:10)
    at DestroyableTransform.Transform.push (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:123:32)
    at DestroyableTransform.sourceMapInit [as _transform] (C:\Users\Jamie\Source\Repos\Ombi\Ombi\Ombi\node_modules\gulp-sourcemaps\src\init\index.js:134:10)
```

**Comment by tidusjar** [comment]
@MattJeanes can you help with this?

**Comment by MattJeanes** [comment]
@tidusjar not run into this personally but I'll take a look when I can

**Comment by MattJeanes** [comment]
Fixed in #1389
